talkative-tangs/Bangazon-LLC-2
# Styling

Description of Proposed Changes
Added bootstrap styling to overall look and feel of site
Added custom styling to navbar

Steps to Test
`git fetch --all`
`git checkout bn-style`

Run through site to ensure all links work as expected.

Impacted Areas in Application
List general components of the application that this PR will affect:
base.html
website.css

Mentions
@ousamasama 
@laboyd001 
@CurtainUp
